src/contexts/ProductsProvider.tsx

```
@@ -0,0 +1,58 @@
+import React, { createContext, PropsWithChildren, useState } from "react";
+import { Category, Error, Product, SortOption } from "../types/product.type";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)